app/views/AmendConfirmationView.scala.html

```diff
@@ -23,32 +24,46 @@
         config: FrontendAppConfig
 )
 
-@(claimId: String)(implicit request: Request[_], messages: Messages)
+@(claimId: String, sections: Seq[Section])(implicit request: Request[_], messages: Messages)
 
 @govukLayout(
     pageTitle = Some(s"${titleNoForm(messages("confirmation.amend.title"))}")
 ) {
 
-    <div class="govuk-panel govuk-panel--confirmation">
-        @html.h1("confirmation.amend.heading", classes="govuk-panel__title")
-        <div class="govuk-panel__body">
-            @messages("confirmation.referenceNumber")<br><strong> @claimId.toString </strong>
+    <div class="govuk-!-display-none-print">
+        <div class="govuk-panel govuk-panel--confirmation">
+            @html.h1("confirmation.amend.heading", classes="govuk-panel__title")
+            <div class="govuk-panel__body">
+                @messages("confirmation.referenceNumber")<br><strong> @claimId.toString </strong>
+            </div>
         </div>
-    </div>
 
-    @html.p{ @messages("amend.confirmation.receivedInformation") }
+        @html.p{ @messages("amend.confirmation.receivedInformation") }
+
+        @components.printPage("amend.confirmation.print", Some("amend.confirmation.print.text"))
 
-    @html.h2("amend.confirmation.whatHappensNext.heading", classes = "govuk-heading-m")
+        <noscript>
+            @html.p {
+                @html.a(
+                    link = AmendConfirmationController.onSummary().url,
+                    messageKey = "amend.confirmation.review",
+                    isExternal = true
+                )
+                @messages("amend.confirmation.print.text")
+            }
+        </noscript>
 
-    @html.p{ @messages("amend.confirmation.amendWhatHappensNext.p1") }
+        @html.h2("amend.confirmation.whatHappensNext.heading", classes = "govuk-heading-m")
 
-    @html.h2("amend.confirmation.moreInformation.heading", classes = "govuk-heading-m")
-    @html.p{ @messages("amend.confirmation.moreInformation.p1") }
+        @html.p{ @messages("amend.confirmation.amendWhatHappensNext.p1") }
+        @html.p{ @messages("amend.confirmation.amendWhatHappensNext.p2") }
 
-    @html.h2("amend.confirmation.helpUs.heading", classes = "govuk-heading-m")
-    @html.p{ @messages("amend.confirmation.helpUs.p1") }
-    @html.p{
-        @html.a(config.feedbackSurvey, messages("amend.confirmation.helpUs.p2"))
-        @messages("amend.confirmation.helpUs.p3")
-    }
+        @html.p{
+            @html.a(config.feedbackSurvey, messages("confirmation.improveService.link.text1"))
+        }
+    </div>
+    <div class="ndrc-!-display-only-print">
+        @html.h1("confirmation.summary.title")
```

Comment:
RED: This is still the wrong key.  Currently the message says: "Your application details"
We are not making an application.  We are making a new application.

Reply:
Yeah but I dont know what it should say o_0